packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -741,7 +744,7 @@ class ToggleButton extends StatelessWidget {
         child: MaterialIconLabel(
           label: text,
           iconData: icon,
-          includeTextWidth: includeTextWidth,
```

Comment:
a perhaps poor judgement call was I changed to an ugly parameter name for this class so that the unscaled width could still be used.